Waterdeep/The Stone of Golorr.md

```diff
@@ -6,3 +6,4 @@ Tags:
 
 - Stolen
 - The Zhentarim believe it was taken by [[Dagult Neverember]]
+- Believed to be used to find [[Renaer Neverember|Renaer's]] father [[Dagult Neverember|Dagult]], treasure
```

Comment:
Typo or something on this line? Don't need to mention Dagult is is Renaer's father here either. It's mentioned on their character pages and in the adventure notes where we learn this fact.